packages/website/lib/subscribe.js

```diff
@@ -0,0 +1,29 @@
+import constants from './constants'
+
+const API = constants.API
+
+/**
+ *
+ * @param {string} email
+ * @param {string} [version]
+ */
+
+export const subscribe = async (email, version = '') => {
+  const loginURL = version
+    ? `/v${version}/internal/blogSubscribe`
+    : '/internal/blogSubscribe'
+  const res = await fetch(API + loginURL, {
```

Comment:
Version stuff is legacy...no need to include here (and can be removed from existing routes).

```suggestion
export const subscribe = async (email) => {
  const subscribeURL = '/internal/blogSubscribe'
  const res = await fetch(API + subscribeURL, {
```

Reply:
I ticketed the removal of the version info from the other routes https://github.com/nftstorage/nft.storage/issues/906